src/hotspot/share/jfr/objectprofiler/objectProfiler.cpp

```diff
@@ -80,8 +81,8 @@ void ObjectProfiler::stop(jlong event_id) {
   OrderAccess::release_store(&_enabled, JNI_FALSE);
 }
 
-jint ObjectProfiler::enabled() {
-  return OrderAccess::load_acquire((volatile jint*)&_enabled);
+int ObjectProfiler::enabled() {
```

Comment:
the declaration of this method didn't be changed.

Reply:
ok